### ISSUE TITLE
Make twisted/conch/unix.py conform to coding standards

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7937]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#7937|
|Type|task|
|Created|2015-06-12 08:19:09Z|
|Branch|https://github.com/twisted/twisted/tree/conch-unix-formatting-7937|

It's glowing bright orange in my emacs, and has ways of writing Python that I didn't know were valid.

<details><summary>Searchable metadata</summary>

```
trac-id__7937 7937
type__task task
reporter__jml jml
priority__normal normal
milestone__ 
branch__branches_conch_unix_formatting_7937 branches/conch-unix-formatting-7937
branch_author__jml jml
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__ 
time__1434097149876391 1434097149876391
changetime__1434125868570212 1434125868570212
version__None None
owner__jml jml
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_2"></a>|@jml commented|
|-|-|

(In [45034]) Branching to conch-unix-formatting-7937.

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_4"></a>|@jml commented|
|-|-|

This branch cleans up many of the warnings from twistedchecker, with the following exceptions:

 * I am not writing documentation, because that's too much work
 * I am not referring to a test module, because no such thing exists
 * I am not capitalizing the first letter of pylint suppression warnings. That's a bug in twistedchecker: https://github.com/twisted/twistedchecker/issues/103

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

Thanks for your changes.

They look good.


I think that I broke the twistedchecker master... will work on fixing it.

--------

This is why I have now

```
$ twistedchecker twisted.conch.unix | grep -v W9208 | grep -v W9202 | grep -v W9002
************* Module twisted.conch.unix
W9402: 88,0: The first letter of comment should be capitalized
W9402:110,0: The first letter of comment should be capitalized
W9015: 84,0: Too many blank lines, found 2
W9015: 85,0: Too many blank lines, found 2
W9015: 86,0: Too many blank lines, found 2


```


-----------

I have no idea if pylint ignores like this one are allowed in Twisted code.

```
# Twistedchecker warns of bad name for these two methods (they should be
# more comments...
# ...
def global_tcpip_forward(self, data):  # pylint: disable=C0103
```


global_tcpip_forward is a valid name as it is mapped from a protocol request (GLOBAL_REQUEST) - https://github.com/twisted/twisted/blob/e0ee4f209ac984b2e465b631e79e717fee882b63/twisted/conch/avatar.py#L33

Rather than adding such junk in our code we should update twistedchecker so that it will accept these methods. ... there is this ticket https://github.com/twisted/twistedchecker/issues/76

Please add your feedback there. Thanks!


I am leaving this in the review queue so that other developers can leave their comments.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_6"></a>|@jml commented|
|-|-|

* I've removed the warning suppression and comments around `global_tcpip_forward`
* Those are the expected warnings

It is my understanding that this needs no further changes and has nothing at all controversial in it.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_7"></a>|@adiroiban commented|
|-|-|

All good.

I have triggered a build on all stable builder for latest commit.

Please merge if all is green.

Twistedchecker is red but is not related to this branch... or the errors are false positives

Here is my run

```
$ twistedchecker twisted.conch.unix | grep -v W9208 | grep -v W9202
************* Module twisted.conch.unix
W9002:  1,0: Missing a reference to test module in header
C0103: 84,4:UnixConchUser.global_tcpip_forward: Invalid name "global_tcpip_forward" for type method (should match ((([a-z])|([a-z]+_[a-z]))[a-zA-Z0-9]+)$|(__[a-z]+__)$|(\_.+)$|(test\_.+)$)
C0103:106,4:UnixConchUser.global_cancel_tcpip_forward: Invalid name "global_cancel_tcpip_forward" for type method (should match ((([a-z])|([a-z]+_[a-z]))[a-zA-Z0-9]+)$|(__[a-z]+__)$|(\_.+)$|(test\_.+)$)

```

**Comment by twisted-trac** [comment]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)<a name="note_8"></a>|@jml set status to `closed`|
|-|-|

(In [45042]) Merge conch-unix-formatting-7937: t/conch/unix.py formatting fixes

Author: jml
Reviewer: adiroiban
Fixes: [#7937](https://github.com/twisted/twisted/issues/7937)

Fixes many PEP 8 and Twisted coding standard violations in
twisted/conch/unix.py
